lib/snapshot/runner.rb

```diff
@@ -167,6 +170,32 @@ def uninstall_app(device_type)
       Helper.backticks("xcrun simctl uninstall #{device_udid} #{Snapshot.config[:app_identifier]} &> /dev/null")
     end
 
+    def add_photos(device_type)
+      Helper.log.debug "Adding photos to #{device_type}..."
+      device_udid = TestCommandGenerator.device_udid(device_type)
+
+      Helper.log.info "Launch Simulator #{device_type}".yellow
+      `xcrun instruments -w #{device_udid} &> /dev/null`
+
+      Snapshot.config[:add_photos].each do |path|
+        Helper.log.info "Adding '#{path}'".yellow
+        `xcrun simctl addphoto #{device_udid} #{path.shellescape} &> /dev/null`
+      end
+    end
+
+    def add_videos(device_type)
```

Comment:
I just noticed those 2 methods are almost the same. Can we just add a parameter for the `type` of media and use the same one?